CHANGE_LOG

```diff
@@ -1,3 +1,30 @@
+Version 0.57.1 (8 June, 2023)
```

Comment:
NOTE: date will need fixing to whenever the tag is made.

Reply:
indeed, we are currently estimating Monday, so will fix that now.

Reply:
IIRC for the 0.57.0 RC release I left it as `TBC`, on the basis that it'll inevitably need patching for the final version. Perhaps the same could be done here?

Reply:
sure, I decided to reverse the order of change-log updates are applied. I am collelcting cherry-picks and updating the change-log in the same PR and will then cherry-pick those changes onto `main` once the FINAL is tagged. It seems much easier this way, especially since I can respond quickly to additional incoming cherry-picks.